client/components/deposits-overview/index.tsx

```diff
@@ -21,15 +22,13 @@ const DepositsOverview = (): JSX.Element => {
 	const { currencies } = overviews;
 
 	const overview = currencies[ 0 ]; // TODO: To handle multiple currencies we'll need to fetch the currently selected currency.
-
+	const currency = 'usd'; // TODO: hardcoded curency for recent deposits.
```

Comment:
Just making a note that this will be updated by @Jinksi PR: https://github.com/Automattic/woocommerce-payments/pull/5822